sandpack-react/src/hooks/useLoadingOverlayState.ts

```diff
@@ -2,7 +2,11 @@ import * as React from "react";
 
 import { useSandpack } from "./useSandpack";
 
-export type LoadingOverlayState = "visible" | "fading" | "hidden";
+export type LoadingOverlayState = "visible" | "fading" | "hidden" | "timeout";
+
+const BUNDLER_TIMEOUT = 300; // 30 seconds timeout for the bundler to respond.
```

Comment:
Shouldn't the comment be `300 ms timeout for the bundler to respond.`?
Also maybe it's a good idea to increase this slightly (maybe 1 second) to account for slower mobile devices and places with slow internet.

Reply:
ah, thanks for spotting that, I ran a test for the second commit and forgot to update back to 30000. I left at 30 seconds for now to be sure there's no latency involved. I've seen scenarios when multiple sandpacks need more than 10 seconds to load on slow connections. So, 30 seconds is safe for a timeout. This should be an extreme case nevertheless.